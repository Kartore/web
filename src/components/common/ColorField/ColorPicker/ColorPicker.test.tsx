import { composeStories } from '@storybook/react';
import { render } from '@testing-library/react';
import { describe } from 'vitest';

import * as Stories from './ColorPicker.stories.tsx';

const { Default } = composeStories(Stories);

describe('Component: ColorPicker', () => {
  describe('Snapshot', () => {
    it('Default', () => {
      const { asFragment } = render(<Default />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
