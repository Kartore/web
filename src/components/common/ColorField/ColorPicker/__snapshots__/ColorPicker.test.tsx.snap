// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Component: ColorPicker > Snapshot > Default 1`] = `
<DocumentFragment>
  <div
    class="h-4 w-4"
  >
    <button
      class="cursor-pointer transition-colors hover:bg-gray-100 active:bg-gray-300 h-full w-full border border-gray-300 focus:outline-0"
      type="button"
    >
      <div
        aria-label="transparent"
        aria-roledescription="color swatch"
        class="h-full w-full"
        id="react-aria-:r0:"
        role="img"
        style="background-color: rgba(255, 255, 255, 0);"
      />
    </button>
  </div>
</DocumentFragment>
`;
